### DISCUSSION TITLE
How to set wrap_results(not wrap for preview window, just the left result lines)?

### DISCUSSION THREAD

**Discussion by onns** [ibhagwan/fzf-lua#1565] [Q&A] [answered]
I'm using LazyVim and v14.x `[fzf-lua](https://github.com/ibhagwan/fzf-lua) as a replacement for [telescope.nvim](https://github.com/nvim-telescope/telescope.nvim)`, How can I see the whole line of the results without `...` ? (project file path is too long to display)

**Comment by ibhagwan** [comment, marked as answer]
See `man fzf`, the default keybind for toggling line wrap is `ctrl-/` or `alt-/` (you can change that in fzf-lua's `keymap.fzf` if you wish).

If you like the line wrap you can add that as a default by adding `fzf_opts =  { ["--wrap"] = true }` to your setup options.

To see more screen space you can also:
> Press `F1` to see all configured binds
- Try fullscreen with `F2`
- Try toggling the preview hide with `F4`
- Rotate the preview with `F5` and `F6` from horizontal to veritcal

I would also recommend trying `:FzfLua files path_shorten=true` (works with any picker) to reduce the paths to something more manageable.
